Edit collection

KhoDis opened the pull request:
![изображение](https://user-images.githubusercontent.com/74288458/227217549-18dc2058-0648-4258-9bc0-e75aa9c54bec.png)
![изображение](https://user-images.githubusercontent.com/74288458/227217586-27032545-a2ca-438f-828c-d5c7261cd972.png)
Возможно, можно будет добавить в BottomAppBar цвет или глаз (hide all/show all), но этих иконок нет в стандартном наборе иконок

Ilya-Sib committed:
feat: Navigation drawer, settings, about us screens added

Ilya-Sib committed:
feat: Navigation drawer, settings, about us screens added

Ilya-Sib committed:
fix: review fixes

Ilya-Sib committed:
fix: review fixes

KhoDis committed:
Initial edit-collection

KhoDis committed:
Initial edit-collection

KhoDis committed:
Fixed Rendering issue

KhoDis committed:
Fixed Rendering issue

rorione commented:
![image](https://user-images.githubusercontent.com/21989227/227249209-0b03e664-5518-4ac3-80e0-df529d398fe9.png)
иконки есть, я в своем мре либу подключал со всеми иконками гугла

rorione committed:
Note options (#5)

* - Note options implemented
- Feed checkboxes added

Feyyus commented:
> ![image](https://user-images.githubusercontent.com/21989227/227249209-0b03e664-5518-4ac3-80e0-df529d398fe9.png) иконки есть, я в своем мре либу подключал со всеми иконками гугла

Ну, я где-то читал, что эта либа очень жирная и лучше эти иконками по необходимости загружать в `res`. Поэтому и смутился

Ilya-Sib committed:
fix: Column replaced with Surface

Ilya-Sib committed:
Merge pull request #6 from Curo-app/settings

Settings

rorione commented:
> > ![image](https://user-images.githubusercontent.com/21989227/227249209-0b03e664-5518-4ac3-80e0-df529d398fe9.png) иконки есть, я в своем мре либу подключал со всеми иконками гугла
> 
> Ну, я где-то читал, что эта либа очень жирная и лучше эти иконками по необходимости загружать в `res`. Поэтому и смутился

вроде можно shrink делать потом и размер станет ок, так что в любом случае пока пользуемся, а потом уже либо в ресурсы положим, либо сделаем shrink

rorione commented:
Кстати, не хватает удаления элемента из коллекции

KhoDis committed:
Added gesture

KhoDis committed:
Added gesture

KhoDis committed:
Added collection renaming

KhoDis committed:
Added collection renaming

KhoDis committed:
Added Stable to class

KhoDis committed:
Added Stable to class

KhoDis committed:
Added workaround for IDE bug

KhoDis committed:
Added workaround for IDE bug

rorione committed:
App composed (#8)

* App composed

- Implemented navigation
- Implemented composition of all screens
- Removed a lot of junk

KhoDis committed:
Merge remote-tracking branch 'origin/edit-collection' into edit-collection

# Conflicts:
#	app/src/main/java/io/github/curo/MainActivity.kt
#	app/src/main/java/io/github/curo/data/NotePreviewModel.kt
#	app/src/main/java/io/github/curo/ui/MainScreen.kt
#	app/src/main/java/io/github/curo/ui/base/Feed.kt
#	app/src/main/java/io/github/curo/ui/screens/Collections.kt
#	app/src/main/res/values/strings.xml

KhoDis committed:
Fixed code after rebase

rorione commented:
Закрыт, т.к. влили вместе с search view
